qtop.py

```diff
@@ -1707,18 +1723,25 @@ def get_batch_system(cmdline_switch, env_var, config_file_batch_option):
                 if options.SAMPLE >= 2:
                     add_to_sample(os.path.join(realpath(QTOPPATH), QTOPCONF_YAML), savepath)
 
+                if scheduler == "pbs":
+                    scheduling_system = PBSBatchSystem(in_out_filenames)
+                elif scheduler == "oar":
+                    scheduling_system = OARBatchSystem(in_out_filenames)
+                elif scheduler == "sge":
+                    scheduling_system = SGEBatchSystem(in_out_filenames)
+
```

Comment:
Yes, something like [this](http://stackoverflow.com/questions/301134/dynamic-module-import-in-python) will be used with a list of modules or classes to be dynamically loaded. But it's not the most urgent thing for right now unless we have the use cases. :)